docs/api-reference/layers/editable-geojson-layer.md

```diff
@@ -207,9 +206,24 @@ The following accessors function the same, but can accept additional arguments:
 
 The additional arguments (in order) are:
 
+* `feature`: the given feature
 * `isSelected`: indicates if the given feature is a selected feature
 * `mode`: the current value of the `mode` prop
 
+### Drawing Feature
+
+While drawing, portion of a feature which has not been "committed" yet can hold its own props. For a LineString, this would be the last line segment moving under the mouse. For a Polygon, this would be the last segment and the fill moving under the mouse. For Rectangles and Circles, this would be the whole feature during drawing. Define the properties with the following accessors:
```

Comment:
What does "while drawing" mean? "while editing"?

As you can imagine, "draw" is very overloaded term in our suite of WebGL frameworks. Any rendering of the layers could be called "drawing", as could the user's act of "drawing" a new line.

The best approach is usually to  carefully choose a terminology and then use it consistently in docs and API design. If "drawing" is used, it may need to be explained somewhere.

E.g could this be called:

```
getEditLineColor
...
```

Reply:
I see what you mean. These props only define a line during a `draw` mode edit, so I was going off that terminology. Maybe they both need a name change, but "edit" seems too broad because these props don't extend to the edits made during `modify`. Perhaps "create" would be better suited for the API?

I've also been calling this a "line hint", but not sure that's totally clear either.

Reply:
Understood. The way we deal with this type of issues in e.g. deck.gl is to have "API audits". 

@georgios-uber Suggest adding these comments to an "API Audit Tracker" issue and then convene the team of developers and decide together, documenting any decisions.